From 9fc32b75251760837db0c1019d0aa8f56a5c4ad2 Mon Sep 17 00:00:00 2001
From: Darshan Jasanee <darshan.jasanee@einfochips.com>
Date: Thu, 12 Sep 2024 11:49:42 +0530
Subject: [PATCH] Update default configuration

Set default buffer size to 8KB in default.pa file.

Upstream-Status: Inappropriate [This is a board specific change]

Signed-off-by: Darshan Jasanee <darshan.jasanee@einfochips.com>
---
 src/daemon/default.pa.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/daemon/default.pa.in b/src/daemon/default.pa.in
index 81dfe7103..85373b1a4 100755
--- a/src/daemon/default.pa.in
+++ b/src/daemon/default.pa.in
@@ -55,7 +55,7 @@ ifelse(@HAVE_MKFIFO@, 1, [dnl
 ### Automatically load driver modules depending on the hardware available
 ifelse(@HAVE_UDEV@, 1, [dnl
 .ifexists module-udev-detect@PA_SOEXT@
-load-module module-udev-detect
+load-module module-udev-detect tsched=1 tsched_buffer_size=8192
 .else
 ], @HAVE_COREAUDIO@, 1, [dnl
 .ifexists module-coreaudio-detect@PA_SOEXT@
-- 
2.25.1

